### PR TITLE
Sidebar & Slider Mods

### PR DESCRIPTION
**Pull request by jrittvo**
This PR proposes the following changes:

1.  Adjust Image2Image STRENGTH slider minimum value down to 0.37 in response to https://github.com/godly-devotion/MochiDiffusion/issues/266

2.  Change STARTING IMAGE label to INPUT IMAGE   <---------- personal preference

3.  Reorder sidebar elements and tweak spacing.  Includes adjusting the length of the STRENGTH slider to match the length of the CONTROLNET drop down, when the sidebar is set at minimum width.  Probably it should "adjust to fill", as the drop down does, but I don't know how to code that.   (Updated it to auto-adjust, thanks to my friend ChatGPT.)   <---------- personal preference
 
No offense taken if my personal preferences don't sit well with your vision of the UI.

Screencap

### CODE REVIEW

**Comment by jrittvo** [comment]
ChatGPT says to change `.frame(width: 110)` to `.frame(maxWidth: .infinity)`.  At least on the third try asking.  The first answer used UIKit and the second used Cocoa.  As little as I do know, I realized those were not right for Swift in macOS.  Giving it a larger piece of the existing code helped set it straight.  I will try the change out sometime tonight.

File: Mochi Diffusion/Views/SidebarControls/StartingImageView.swift [hunk 1]
```diff
@@ -61,9 +61,9 @@ struct StartingImageView: View {
                 )
                 .sidebarLabelFormat()
 
-                Slider(value: $controller.strength, in: 0.685...1, step: 0.035)
+                Slider(value: $controller.strength, in: 0.37...1, step: 0.07)
                     .controlSize(.mini)
-                    .frame(width: 88)
+                    .frame(maxWidth: .infinity)
```
Comment (godly-devotion) [inline Mochi Diffusion/Views/SidebarControls/StartingImageView.swift:66]: Good catch on the UI here. Removing the `.frame` modifier gives the same results.

**Comment by godly-devotion** [comment]
Thanks for your PR. For now I think the current control order & wording is appropriate.
However I remember on Discord that the starting image strength value range was too high, I think that is a good change to have.

**Comment by jrittvo** [comment]
You're welcome to use any parts of the PR you choose.  You can just use the StartingImageView.swift file, and only the slider change(s) there.  I know rewording opens up the need for a new translations round, which makes no sense at all for 1 word.

File: Mochi Diffusion/Views/SidebarControls/StartingImageView.swift (hunk 1 above)
Comment (jrittvo) [reply Mochi Diffusion/Views/SidebarControls/StartingImageView.swift:66]: So the default behavior is to auto adjust?  Good to know!

**Comment by godly-devotion** [comment]
I can make the changes if its easier for you. If you want, you could try giving it a shot in this PR as a git exercise ;)

**Comment by jrittvo** [comment]
Please make the changes.  It takes me at least 2 tries in git to get things submitted right.  I think I know how to use branches now, which will make it easier in the future, but this one is still all over the place in my repos.  I'm actually looking forward to rebasing my repo to before I submitted the PR and then doing a git pull to clean out the mess I made.

I am trying to understand the code in the Image Preview PR right now.  That will keep me busy for a while.  ml-stable-diffusion added a boolean `public var` for `useDenoisedIntermediates`.  So I think the only change to the PR for Mochi would be to set that variable to true.  It defaults to false, which I what I saw it doing last night.  But first I need to learn how to set a `public var`.  Baby steps.

**Comment by godly-devotion** [comment]
Implemented via 2f8134d

**Pull request closed without merging**